docs/commands/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _commands-index:
+
+
+index
+=========
+
+Index a list of haplotypes.
+
+The ``index`` command takes as input a list of haplotypes (specified as a :doc:`.hap file </formats/haplotypes>`) and outputs the list as a .gz and a .tbi file.
+
+Usage
+~~~~~
+.. code-block:: bash
+
+	haptools index \
+	--output PATH \
+	--verbosity [CRITICAL|ERROR|WARNING|INFO|DEBUG|NOTSET] \
+	HAPLOTYPES
+
+Example
+~~~~~~~~
+.. code-block:: bash
+
+	haptools index tests/data/basic.hap
+
```

Comment:
might also be good to add an example depicting usage of the output parameter. Maybe something like this?
```suggestion
You may also specify a custom output path.
.. code-block:: bash

	haptools index --output tests/data/sorted.basic.hap.gz tests/data/basic.hap

```